lib/tests/facts/schema.cc

```diff
@@ -154,6 +155,20 @@ struct kernel_resolver : resolvers::kernel_resolver
     }
 };
 
+struct ldom_resolver : resolvers::ldom_resolver
+{
+protected:
+    virtual data collect_data(collection& facts) override
+    {
+        data result;
+        ldom_info single_value;
+        single_value._key = "domainname";
```

Comment:
I still don't understand where the definition for ldom_info is coming from here. `_key` should fail to compile.